core/src/stored/spool.cc

```diff
@@ -298,7 +299,10 @@ static bool DespoolData(DeviceControlRecord* dcr, bool commit)
   rdev->min_block_size = dcr->dev->min_block_size;
   rdev->device_resource = dcr->dev->device_resource;
   rdcr = dcr->get_new_spooling_dcr();
-  SetupNewDcrDevice(jcr, rdcr, rdev.get(), NULL);
+  struct BlockSizeBoundaries devboundries {
+    dcr->dev->max_block_size, dcr->dev->min_block_size
+  };
+  SetupNewDcrDevice(jcr, rdcr, rdev.get(), &devboundries);
```

Comment:
If you look into `SetupNewDcrDevice()` you'll see that passing the fourth parameter just does the same thing this code already did on lines 298-299.